packages/core/src/Modules/AccountDeactivated/account-deactivated.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Modal } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { getDerivComLink } from '@deriv/shared';
+
+const AccountDeactivated = ({ logout }) => {
+    const [is_modal_open, setModalState] = React.useState(true);
+    const [timer, setTimer] = React.useState(10);
+
+    React.useEffect(() => {
+        window.history.pushState(null, null, '/');
```

Comment:
please use useHistory from react-router-dom -> routes 🙏

Reply:
useHistory update the page immediately which is not good here.
I just want to update the URL in case the user closes the modal and wants to use the login button from the navbar.
if I don't change the url he will redirect to this page after login.